tests/e2e.test.js

```diff
@@ -255,7 +255,7 @@ describe('Search', () => {
     await $searchInput.press('Backspace');
 
     const $gridItemIfEmpty = await page.$('.grid-item--if-empty');
-    expect(await isHidden($gridItemIfEmpty)).toBeTruthy();
+    expect(await isHidden($gridItemIfEmpty)).toBeFalsy();
```

Comment:
Why did this need to be changed?

Reply:
It didn't need to. That's a debug change that I forgot to rollback, and was hiding an error.
Good catch!